### DISCUSSION TITLE
Can this be used as a replacement for the horrifically bad GoPro webcam software?

### DISCUSSION THREAD

**Discussion by whosjohn** [gopro/OpenGoPro#365] [Ideas]
The GoPro webcam software is useless. I mean that. USELESS. Lags like crazy and 0 features. 

Unusable. 

This is baffling in 2023 but here we are. Can you make this compatible with OBS and so on. Would be a game changer not needing capture cards for every GP.

**Comment by KonradIT** [comment]
The GoPro Webcam software merely reencodes the UDP frames which are also accessible via OpenGoPro into a native video virtual driver device. You're free to program an alternative. Personally on Linux I use ffmpeg.

**Reply by whosjohn** [reply]
I wish I was savvy enough to code something like that. I leave that to the pros lol. The driver / UI they gave it is so bad. Its worse than a 2007 webcam I have in my closet.

**Reply by KonradIT** [reply]
Why would the UI matter at all? It's a webcam software, you don't have to use its UI at all. Here's the ffmpeg command, maybe it works better than the official software https://github.com/jschmid1/gopro_as_webcam_on_linux

**Comment by KonradIT** [comment]
GoPro Webcam is compatible with OBS, too.
